examples/basic/alu_hier.py

```diff
@@ -3,6 +3,7 @@
 from torii      import Elaboratable, Module, Signal
 from torii.back import verilog
 
+
```

Comment:
Which of two styles are we going to here? The bulk of these we think were single whitespace between imports and code and this is our vote as we find too many blank lines between things unhelpful and makes parsing visually harder.

If you're really insistent on the extra whitespace, then it's not being consistently applied even in this PR.

Reply:
It is applied consistently, the thing is that `__all__` is part of the import sets *technically* so it gets the same one-line treatment when sitting with the `imports` causing the double new line to be below it.

This is a flake8/isort thing